src/commands/deployments/viewCommitInGitHub.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DeploymentTreeItem } from "vscode-azureappservice";
+import { ext } from "../../extensionVariables";
+
+export async function viewCommitInGitHub(node?: DeploymentTreeItem): Promise<void> {
+    if (!node) {
+        node = <DeploymentTreeItem>(await ext.tree.showTreeItemPicker('deployment/github'));
+    }
+    await node.viewCommitInGitHub();
```

Comment:
I can't see where this is implemented. Any pointers?

Reply:
https://github.com/Microsoft/vscode-azuretools/blob/v0.33.0-appservice/appservice/src/tree/DeploymentTreeItem.ts#L134-L144